docs/full-tutorial.md

```diff
@@ -809,4 +807,4 @@ You can see that the name is `CasperTest` in this example.
  }
 }
 
-```
\ No newline at end of file
+```
```

Comment:
Are we just removing the previous text or adding new one too?

Reply:
Just updated to include a note on the new system.